cypress/e2e/widgets/openshift-rhel.cy.ts

```diff
@@ -48,6 +35,6 @@ describe('Red Hat OpenShift Widget', () => {
   it('disappears when removed from the layout', () => {
     cy.resetToDefaultLayout();
     cy.get(`[data-ouia-component-id="${widgetId}"]`).should('be.visible');
```

Comment:
@catastrophe-brandon not blocking - but I wonder in the future if we always scroll the widget being tested into view when performing these tests (like we do in openshift-ai.cy.ts). Then in another test we can have an assertion to make sure the default layout has certain widgets in view (without scrolling)?

Reply:
We may need an OUIA attribute elsewhere in the widget. When I've used scrolling on these, it tends to scroll past the widget header, leading to failures later in the test.

Reply:
Odd... That could also be related to the library we are using to render the grid.